frontend/src/app/search/SearchForm.tsx

```diff
@@ -20,20 +21,25 @@ export function SearchForm({ initialSearchResults }: SearchFormProps) {
     initialSearchResults,
   );
 
+  const formRef = useRef(null);
```

Comment:
ref allows us to submit from non submit type inputs (which are basically all our inputs since we just have one submit/search button in the `SearchBar` component